cookbook_se/recipes/core/README.rst

```diff
@@ -1,16 +1,7 @@
 ############################################
 Getting Started with Flytekit [python]
 ############################################
-
-The intention of this cookbook is to provide a hands on tutorial of various
-features in Flyte (specifically if you are using flytekit-python). Flytekit is
-intended to be useful even without Flyte backend. All the examples in the
-following set are executable locally and we recommend the best way to try out
-Flyte is to run these examples locally. The tutorial is divided into 4 sections
-
-``Basic``, ``Intermediate`` & ``Advanced`` sections are laid out in varying levels of difficulty.
-``Working with Hosted Flyte Service`` is an independent section and can be used with any of the difficulty
-level sections.
+The first three chapters of this part of the cookbook is organized into increasing levels of complexity. The last chapter shows the reader the optimal way to apply flytekit to a live Flyte deployment.
```

Comment:
nit: what is "apply flytekit to a live deployment" mean? can we just say "use"?

Reply:
can you make a suggestion?

Reply:
done